LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Pelagicore
+Copyright (c) 2016-2017 Pelagicore
```

Comment:
I'd put this (and the other same change) to a separate patch but it's not a biggie since it's kinda related.